index.html

```diff
@@ -25,13 +25,65 @@
 
     <div id="popUp" class="popUp">
-      <button id="cancel-icon">&times;</button>
+      <button id="cancel-icon" type="button">&times;</button>
       <ul class="menu">
         <li class="item"><a href="#projects">Portfolio</a></li>
         <li class="item"><a href="#about">About</a></li>     
         <li class="item"><a href="#contact">Contact</a></li>
       </ul>
     </div>
+
+
+ <div class="overlay-container">
+  <div id="controller" class="project-details-overlay-outer">
+    <div class="project-details-overlay">
+      <div class="popup-header">
+        <h1 id="topic">Tonic</h1>
+        <button id="exit-btn" type="button" class="popup-btn">&times;</button>
+      </div>
+      <div class="popup-content">
+      <ul class="data nav justify-content-start d-flex">
+        <li id="names" class="canopy">CANOPY</li>
+        <li id="position" class="list-item text">Back-End dev</li>
+        <li id="year" class="list-item text">2022</li>
+      </ul>
+      </div>
+      <div class="popup-img"> 
+        <img id="newImg" class="popup-img" src="media/SnapshootPortfolio1.png" alt="project image">
+      </div>
+      <div class="popup-items"> 
+        <p id="popup-text">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Non, nemo molestias officiis laborum 
+          nisi, nulla magni aspernatur itaque natus dolorem suscipit accusamus error perspiciatis est 
+          nesciunt. Repellendus ullam deleniti dolorem aliquam dicta.
+          orem ipsum dolor sit amet consectetur, adipisicing elit. Non, nemo molestias officiis laborum 
+          nisi, nulla magni aspernatur itaque natus dolorem suscipit accusamus error perspiciatis est 
+          nesciunt. Repellendus ullam deleniti dolorem aliquam dicta.
+          orem ipsum dolor sit amet consectetur, adipisicing elit. Non, nemo molestias officiis laborum 
+          nisi, nulla magni aspernatur itaque natus dolorem suscipit accusamus error perspiciatis est 
+          nesciunt. Repellendus ullam deleniti dolorem aliquam dicta.
+        </p>
+      <div class="popup-tech">
+        <ul class="popup-ul">
+          <li class="html-btn"><img src="media/html.png" alt="html" /></li>
+          <li class="html-btn"><img src="media/css.png" alt="css" /></li>
+          <li class="html-btn"><img src="media/Js.png" alt="javaScript"></li>
+        </ul>
+        <ul class="popup-ul">
+          <li class="html-btn"><img src="media/Rubyonrails.png" alt="Ruby on rails"/></li>
+          <li class="html-btn"><img src="media/css.png" alt="css" /></li>
+          <li class="html-btn"><img src="media/Js.png" alt="javaScript"></li>
+        </ul>
+        <div>
+          <button type="button" class="big-btn">See Live<span><img src="media/Icon.svg" alt=""></span></button>
+          <button id="github" type="button" class="big-btn">See Source<span><img id="git" src="media/Icon-GitHub.svg" alt=""></span></button>
+        </div>
+      </div>
+      </div>
+ 
+    </div>
+  </div> 
+  </div>
```

Comment:
- Kindly try to match the Figma design for the list of technologies
![image](https://user-images.githubusercontent.com/24703171/156663139-33ff0a79-fd99-407d-9613-71e6b391a3fe.png)
the `ruby on rails` technology doesn't have a background.
- Kindly try to match the Figma design for the ` see live ` and `see source` buttons. Please increase the spacing between the icon and the text and make the `see button` icon bigger.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/156663432-71d8169b-87eb-40fb-be3d-b6ffa34c9977.png) | ![image](https://user-images.githubusercontent.com/24703171/156663544-2ec968a5-c86f-450a-9e05-c24e68298510.png) |

- Kindly try to match the Figma design in the desktop version for the `see live` and `see buttons` in width `992px`. Please make the buttons inside each other and with the same width and height

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/156664246-fa233a02-46c7-4d30-97ee-410a2b1eef2a.png) | ![image](https://user-images.githubusercontent.com/24703171/156664410-279ea4dd-b2f5-4663-9461-06390ce156a8.png) |

- Kindly try to open the modal from the first time when I press the `see project` button because the first time I load the page I must press the `see project` button three times to open but after that, it works fine.
- Please remove the red border that surround the modal in the mobile version to match the Figma design.